src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testBoiler/BoilerTest.java

```diff
@@ -1,12 +1,89 @@
 package org.fundacionjala.prog101CoffeeMaker.coffeeMaker.testBoiler;
 
+import org.junit.Assert;
 import org.junit.Test;
 
 import static org.junit.Assert.*;
 
 public class BoilerTest {
 
     @Test
-    public void containLiquid() {
+    public void containLiquid_ThereIsNoLiquid_False() {
+        Boiler boiler = new Boiler();
+        boolean expected = false;
+        boolean actual = boiler.containLiquid();
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void containLiquid_ThereIsOneCoupOfLiquid_True() {
+        Boiler boiler = new Boiler();
+        boiler.setCupsOfWater(1);
+        boolean expected = true;
+        boolean actual = boiler.containLiquid();
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void containLiquid_ThereIsTwelveCoupsOfLiquid_True() {
+        Boiler boiler = new Boiler();
+        boiler.setCupsOfWater(12);
+        boolean expected = true;
+        boolean actual = boiler.containLiquid();
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void on_HeaterOn_True() {
+        Boiler boiler = new Boiler();
+        boiler.on();
+        boolean expected = true;
+        boolean actual = boiler.getWorkingBoiler();
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void off_HeaterOff_False() {
+        Boiler boiler = new Boiler();
+        boiler.off();
+        boolean expected = false;
+        boolean actual = boiler.getWorkingBoiler();
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void getCupsOfWater_Zero() {
+        Boiler boiler = new Boiler();
+        boiler.setCupsOfWater(0);
+        int expected = 0;
+        int actual = boiler.getCupsOfWater();
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void getCapacity_Twelve() {
+        Boiler boiler = new Boiler();
+        boiler.setCupsOfWater(12);
+        int expected = 12;
+        int actual = boiler.getCupsOfWater();
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void setCapacity_NewCapacityZero_Zero() {
```

Comment:
the name of the method is glasses of water, but the test indicates capacity, it is inconsistent

Reply:
that's correct. I changed it edson